src/System.Private.CoreLib/shared/System/Globalization/EastAsianLunisolarCalendar.cs

```diff
@@ -256,46 +244,37 @@ public override DateTime ToDateTime(int year, int month, int day, int hour, int
                             SR.Format(SR.ArgumentOutOfRange_Day, daysInMonth, month));
             }
 
-            int gy = 0; int gm = 0; int gd = 0;
+            int gy = 0;
+            int gm = 0;
+            int gd = 0;
 
-            if (LunarToGregorian(year, month, day, ref gy, ref gm, ref gd))
-            {
-                return new DateTime(gy, gm, gd, hour, minute, second, millisecond);
-            }
-            else
+            if (!LunarToGregorian(year, month, day, ref gy, ref gm, ref gd))
             {
                 throw new ArgumentOutOfRangeException(null, SR.ArgumentOutOfRange_BadYearMonthDay);
             }
-        }
 
+            return new DateTime(gy, gm, gd, hour, minute, second, millisecond);
+        }
 
-        //
         // GregorianToLunar calculates lunar calendar info for the given gregorian year, month, date.
         // The input date should be validated before calling this method.
-        //
         internal void GregorianToLunar(int nSYear, int nSMonth, int nSDate, ref int nLYear, ref int nLMonth, ref int nLDate)
         {
-            //    unsigned int nLYear, nLMonth, nLDate;    // lunar ymd
-            int nSolarDay;        // day # in solar year
-            int nLunarDay;        // day # in lunar year
-            int fLeap;                    // is it a solar leap year?
-            int LDpM;        // lunar days/month bitfield
-            int mask;        // mask for extracting bits
-            int nDays;        // # days this lunar month
-            int nJan1Month, nJan1Date;
-
-            // calc the solar day of year
-            fLeap = GregorianIsLeapYear(nSYear);
-            nSolarDay = (fLeap == 1) ? DaysToMonth366[nSMonth - 1] : DaysToMonth365[nSMonth - 1];
+            bool isLeapYear = GregorianIsLeapYear(nSYear);
+            int nJan1Month;
+            int nJan1Date;
```

Comment:
In some cases it looks like you've removed the 'm' or 'n' or such prefix.  In what situations are we removing them and in what situations are we keeping them?

Reply:
Fixed the remaining ones i could find